File: 프로그래머스/unrated/181931. 등차수열의 특정한 항만 더하기/등차수열의 특정한 항만 더하기.java
class Solution {
    public int solution(int a, int d, boolean[] included) {
        int answer = 0;
        
        for(int i = 0; i < included.length; i++) {
			if(included[i] == true) {
				answer += a + (d * i);
			} // included의 원소 값이 true일시 answer에 등차 수열의 값을 더해준다
		}
        return answer;
    }
}

File: 프로그래머스/0/181931. 등차수열의 특정한 항만 더하기/등차수열의 특정한 항만 더하기.js
function solution(a, d, included) {
    var answer = 0;
    
    for (let i = 0; i < included.length; i++) {
        if (included[i] === true) {
            answer += a + (d * i);
        } // included의 원소 값이 true일시 answer에 등차 수열의 값을 더해준다
    }
    return answer;
}